Welolo/client/src/components/TRANSACTION_FORM.js

```diff
@@ -166,115 +163,76 @@ class TRANSACTION_FORM extends Component {
+            <div className="label_wrapper">
+              <label id="recipient_name_label" htmlFor="recipient_name">Recipient Full Name</label>
             </div>
-        );
-      }
-      else
-      {
-        return (
-          <div>
-              <form onSubmit={this.onSubmit}>
-                  <h1>Payment</h1>
-                  <div>
-                      <label htmlFor="recipient_name">Recipient Name:</label>
-                      <input 
-                          type="text"
-                          name="recipient_name"
-                          id="recipient_name"
-                          value={this.state.message.recipient_name}
-                          onChange={this.onHandleChange}
-                      />
-                  </div>
-                  <div>
-                      <label htmlFor="recipient">Recipient Phone Number (ex: +12223334444):</label>
-                      <input
-                          type="tel"
-                          name="recipient"
-                          id="recipient"
-                          value={this.state.message.recipient}
-                          onChange={this.onHandleChange}
-                      />
-                  </div>
-                  <div>
-                      <label htmlFor="sender_quantity">Quantity (USD):</label>
-                      <input
-                          type="text"
-                          name="sender_quantity"
-                          id="sender_quantity"
-                          value={this.state.message.sender_quantity}
-                          onChange={this.onHandleChange}
-                          readOnly
-                      />
-                  </div>
-                  <div>
-                      <label htmlFor="body">Message:</label>
-                      <textarea 
-                          name="body"
-                          id="body"
-                          value={this.state.message.body}
-                          onChange={this.onHandleChange}
-                      />
-                  </div>
-                  <button type="submit" disabled={this.state.submitting}>
-                      Send message
-                  </button>
-              </form>                     
-              <div>{this.state.error ? <p>Please try again! phone number you entered was not a valid phone number</p> : <p></p>}</div>
+            <input 
+              type="text"
+              name="recipient_name"
+              id="recipient_name"
+              value={this.state.message.recipient_name}
+              onChange={this.onHandleChange}
+            />
           </div>
-      );
-      }
-    }
+          <div id="payment_form_recipient_number_wrapper">
+            <div className="label_wrapper">
+              <label id="recipient_number_label" htmlFor="recipient">Recipient Phone Number</label>
+              <div id="recipient_number_label_example">&nbsp;ex: +12223334444 or 2223334444</div>
```

Comment:
Can you remove the '&nbsp;' in TRANSACTION_FORM.js since the hint is on the next line now?